src/components/App.js

```diff
@@ -5,8 +5,10 @@ import Display from './Display';
 export default function App() {
```

Comment:
**REQUIRED** 
* calculate.js is imported in the App component.
![Screenshot from 2021-03-25 16-36-18](https://user-images.githubusercontent.com/44072711/112540602-c1b70d80-8d88-11eb-8cf0-98ee8e06e72f.png)